### PR TITLE
SISRP-41772 - Remove/replace 'testext' tests v2

### PR DESCRIPTION
**Pull request by redconfetti**
https://jira.berkeley.edu/browse/SISRP-41772

### CODE REVIEW

**Comment by redconfetti** [comment]
@raydavis @johncrossman - I'm not sure if I'm removing TestExt tests you think should be retained for manual use (from local command line). Let me know if I went overboard here.

**Comment by raydavis** [comment]
Thanks, @redconfetti . It's true that a lot of OEC and Google support is currently only exercised in "testext" mode, but personally I'm OK with ETS bringing optional versions of such tests back on an as-needed basis.

File: app/models/my_academics/my_academic_roles.rb [hunk 1]
```diff
@@ -59,5 +57,9 @@ def map_roles(default_roles, assigned_roles)
         end
       end
     end
+
+    def term_cpp
+      MyAcademics::MyTermCpp.new(@uid).get_feed
```
Comment (lyttam) [inline app/models/my_academics/my_academic_roles.rb]: @redconfetti do you want to assign this to a class instance variable?
```@term_cpp = MyAcademics::MyTermCpp.new(@uid).get_feed```
That way we aren't creating a new instance of `MyAcademics::MyTermCpp` every time this method is called.

**Comment by lyttam** [comment]
@redconfetti Very nice. Just one small suggestion.

File: app/models/my_academics/my_academic_roles.rb (hunk 1 above)
Comment (redconfetti) [reply app/models/my_academics/my_academic_roles.rb]: @lyttam - Good call

**Pull request merged**